Core/GDCore/Extensions/Builtin/VariablesExtension.cpp

```diff
@@ -143,6 +169,30 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsVariablesExtension(
       .AddParameter("scenevar", _("Variable"))
       .UseStandardOperatorParameters("string");
 
+  extension
+      .AddAction(
+          "SetSceneVariableAsBoolean",
+          _("Boolean value of a scene variable"),
+          _("Modify the boolean value of a scene variable."),
+          _("Set the boolean value of scene variable _PARAM0_ to _PARAM1_"),
+          _("Variables"),
+          "res/conditions/var24.png",
+          "res/conditions/var.png")
+      .AddParameter("scenevar", _("Variable"))
+      .AddParameter("trueorfalse", _("New Value:"));
+
+  extension
+      .AddAction("ToggleSceneVariableAsBoolean",
+                 _("Toggle boolean value of a scene variable"),
+                 _("Toggle the boolean value of a scene variable.\n"
+                   "If it was true, it will become false, and if it was false "
```

Comment:
nitpicking: I recommend putting "If it was true blabla" into its own `_("...")`, so that for translators they have to translate it only once.
Put "\n" outside of a translation marker.